charts/quickwit/values.yaml

```
@@ -426,6 +426,11 @@ prometheusRule:
 service:
   type: ClusterIP
 
+  # -- Set the ip family policy to configure dual-stack see [Configure dual-stack](https://kubernetes.io/docs/concepts/services-networking/dual-stack/#services)
+  ipFamilyPolicy: ""
+  # -- Sets the families that should be supported and the order in which they should be applied to ClusterIP as well. Can be IPv4 and/or IPv6.
+  ipFamilies: []
+
```

Not sure what the best practice is here.

```suggestion
  # -- Set the ip family policy to configure dual-stack. See [Configure dual-stack](https://kubernetes.io/docs/concepts/services-networking/dual-stack/#services)
  # ipFamilyPolicy: "PreferDualStack"
  # -- Sets the families that should be supported and the order in which they should be applied to ClusterIP as well. Can be IPv4 and/or IPv6.
  # ipFamilies: ["IPv4","IPv6"]

```

---

We could comment it out to indicate that it's not a required configuration if you prefer. However, I believe it's better to retain the current value to demonstrate the default behavior.

---

the main benefit of commenting out I think is that when we test it, we test that it is compatible with old configuration that didn't have the value! 

> I believe it's better to retain the current value to demonstrate the default behavior

What is the default behavior actually? `""` and `[]` are pretty confusing to me.

---

> the main benefit of commenting out I think is that when we test it, we test that it is compatible with old configuration that didn't have the value!
As you want :)

> What is the default behavior actually? "" and [] are pretty confusing to me.
They are not set as they are treated as empty value for gotemplate and use default kubernetes values.

The default kubernetes values are : 
ipFamilyPolicy: "SingleStack"
ipFamilies: "" # no value, it depends on cluster configuration and ipFamilyPolicy

https://kubernetes.io/docs/reference/kubernetes-api/service-resources/service-v1/